scala-api/src/main/scala/com/cloudera/livy/scalaapi/ScalaJobContext.scala

```diff
@@ -26,21 +26,47 @@ import org.apache.spark.streaming.StreamingContext
 
 import com.cloudera.livy.JobContext
 
+/**
+ *  Holds runtime information about the job execution context.
+ *
+ *  An instance of this class is kept on the node hosting a remote Spark context and is made
+ *  available to jobs being executed via RemoteSparkContext#submit().
+ *
+ *  Wrapper over the java JobContext of livy
```

Comment:
Also, like the other case, the Java API is not exposed so this info is not interesting.